Formula/trader.rb

```diff
@@ -0,0 +1,29 @@
+class Trader < Formula
+  desc "Star Traders"
+  homepage "https://www.zap.org.au/projects/trader/"
+  url "https://ftp.zap.org.au/pub/trader/unix/trader-7.13.tar.xz"
+  sha256 "0d2b51134166b0f436dc6423e2ce378b1df929a9de141c002f3da86af18bb262"
+
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
+  depends_on "ncurses" # The system version does not work correctly
+
+  def install
+    ENV.prepend_path "PKG_CONFIG_PATH",
+        Formula["ncurses"].opt_libexec/"lib/pkgconfig"
+    args = %W[
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --prefix=#{prefix}
+      --with-libintl-prefix=#{Formula["gettext"].opt_prefix}
+    ]
+    system "./configure", *args
+    system "make", "install"
+  end
+
+  test do
+    # Star Traders is an interactive game, so the only option for testing
+    # is to run something like "trader --version"
+    system "#{bin}/trader", "--version"
```

Comment:
Thanks for the PR, the install looks good on the first try 👍. I just have a comment on the testing.

We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I had read those comments in the Guide beforehand.  Unfortunately, the game doesn't have command-line functionality apart from "--help" and "--version".  However, I did think of something like this:

```
test do
    assert_equal "Star Traders (trader) #{version}" shell_output("env LC_ALL=C LANGUAGE=en #{bin}/trader --version").lines.first.chomp
```

Would that be sufficient?

Reply:
Just to reemphasise: the *only* non-GUI (ie, non-interactive) functionality that the program has is to display a response to `--help` and `--version`.  Everything else brings up the interactive game.

Reply:
I see, that makes sense.